code/core/src/main/java/com/adobe/marketing/mobile/services/ui/vnext/Presentation.kt

```diff
@@ -39,3 +42,19 @@ class InAppMessage(
      */
     lateinit var eventHandler: InAppMessageEventHandler
 }
+
+/**
+ * Represents a FloatingButton presentation.
+ * @param eventListener the listener for the getting notified about FloatingButton lifecycle events
+ * @param settings the settings for the FloatingButton
+ */
+class FloatingButton(
+    val settings: FloatingButtonSettings,
+    val eventListener: FloatingButtonEventListener
+) : Presentation<FloatingButton>(eventListener) {
+
+    /**
+     * The event handler for the FloatingButton.
+     */
+    lateinit var eventHandler: FloatingButtonEventHandler
```

Comment:
Can you also add `internal set` visibility modifier and make a similar update for InAppMessageEventHandler?

Reply:
Yes good point! Will add it.